pkg/loop/internal/service.go

```diff
@@ -11,6 +12,8 @@ import (
 	"github.com/smartcontractkit/chainlink-relay/pkg/services"
 )
 
+var ErrPluginUnavailable = errors.New("plugin unavailable")
```

Comment:
This seems OK to export from `package loop`, and potentially even useful. I guess we need to be able to reference it from inside here though? Maybe another var to forward in that case? :shrug: